apps/molgenis-components/src/components/forms/baseInputs/BaseInput.vue

```diff
@@ -64,5 +64,6 @@ export default {
         : this.label;
     },
   },
+  emits: ["update:modelValue"],
```

Comment:
"Interesting curiosity" Maybe a sign of something broken:
The ButtonAction fires 2 triggers in the preview
https://preview-emx2-pr-1778.dev.molgenis.org/apps/molgenis-components/#/component/ButtonAction

But One trigger here: https://emx2.dev.molgenis.org/apps/molgenis-components/#/component/ButtonAction

Reply:
interesting indeed. In vue3 the alert is treated as a child element and the click on the 'ok' of the alert propagates also to the parent element. Now I fixed that using the .self modifier. I can imagine more issues like this to emerge

Reply:
However, I don't understand why this is an issue in ButtonAction and not in ButtonAlt

Reply:
Hmm, it is not even consistent. Because now I changed it back and still works allright.